src/components/NewWorkspaceModal.js

```diff
@@ -111,7 +111,12 @@ const NewWorkspaceModal = withDisplayName('NewWorkspaceModal', ({
         }],
         async () => {
           const workspace = await Ajax().Workspaces.create(body)
-          Ajax().Metrics.captureEvent(Events.workspaceCreate, extractWorkspaceDetails(
+          // Only invoke Leo if we are within an Azure Workspace
+          if (!workspace.googleProject) {
+            await Ajax().Leo.create(workspace.name, workspace.id)
```

Comment:
(I know this is still in draft) - the appName here should be "cbas-wds-default" (which should probably move to a constant somewhere) so `getWdsUrl ` can easily find it: https://github.com/DataBiosphere/terra-ui/blob/67469562e6de0d629b6fd8d67427ba2a915df690/src/libs/ajax/data-table-providers/WdsDataTableProvider.ts#L89

Reply:
Is `cbas-wds-default` a placeholder value for now? Or is this something we want to name all WDS apps for the foreseeable future?